src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
Ok, got it. It's for installation of during development.
I feed it's another reason to drop VERSION_MAJOR/MINOR/REVISON/NUMBER since it's does not support "dev0" suffix.